.github/workflows/deploy-docs.yml

```diff
@@ -1,62 +0,0 @@
-# Workflow for building and deploying Flagship Code™ Docs to GitHub Pages
```

Comment:
@crherman7 I think we need to keep our workflows for deploy docs and running test for PRs

Reply:
correct

Reply:
updated